http-transport-binding.md

```diff
@@ -163,16 +163,23 @@ Except for attributes [explicitly handled in this specification]
 HTTP header mapping of well-known CloudEvents attributes is that 
 each attribute name MUST be prefixed with "CE-".
 
+Note: per the [HTTP](https://tools.ietf.org/html/rfc7230#section-3.2)
+specification, header names are case-insensitive. Upper-case letters are
+prefixed with a dash ("-"). When converting a HTTP Header Name back, the dash
+converts the following letter back to upper-case, all other letters are
+lower-case.
+
 Examples:
 
-    * `eventTime` maps to `CE-EventTime`
-    * `eventID` maps to `CE-EventID`
-    * `cloudEventsVersion` maps to `CE-CloudEventsVersion`
+    * `eventTime` maps to `CE-Event-Time`
+    * `eventID` maps to `CE-Event-I-D`
```

Comment:
This one isn't beautiful - does someone has a good suggestion for mapping abbreviations? Alternatively, we could define an exception for `eventID`.

Reply:
yup - that's a real concern and I'm not thrilled with pretty much defining one list for the lifetime of v1.0.  So, `Event-I-D` might be the best choice if we do head down this path - if nothing else its at least consistent, which I do like :-)